src/background/target-page-controller.ts

```diff
@@ -194,14 +168,6 @@ export class TargetPageController {
     }
 
     private sendTabVisibilityChangeAction(tabId: number, isHidden: boolean): void {
-        if (!this.hasTabContext(tabId)) {
-            return;
-        }
-        const tabContext = this.targetPageTabIdToContextMap[tabId];
-        if (tabContext == null) {
```

Comment:
Not sure if this code is just really old or not, but here we used to be checking *both* that the tabID was a key in the map (in the first if statement) *and* that the key's value is non-null. This makes me think that we should be doing the same thing in the new code as well, but it doesn't appear that we are right now.